accounts/repository.go

```diff
@@ -0,0 +1,51 @@
+package accounts
+
+import (
+	"github.com/centrifuge/go-centrifuge/storage"
+)
+
+const accountPrefix string = "account-"
```

Comment:
To be consistent with all the db prefixes, I would change it to:
```suggestion
const accountPrefix string = "account_"
```